tokens/properties/components/Button.json

```diff
@@ -87,6 +87,64 @@
                 }
             }
         },
+        "progressive": {
+            "color": {
+                "value": "{font.color.inverted.value}"
+            },
+            "border-color": {
+                "value": "{border.color.progressive.default.value}"
+            },
+            "background-color": {
+                "value": "{background.color.progressive.default.value}"
+            },
+            "hover": {
+                "border-color": {
+                    "value": "{border.color.progressive.hover.value}"
+                },
+                "background-color": {
+                    "value": "{background.color.progressive.hover.value}"
+                }
+            },
+            "active": {
+                "border-color": {
+                    "value": "{border.color.progressive.active.value}"
+                },
+                "background-color": {
+                    "value": "{background.color.progressive.active.value}"
+                }
+            },
+            "focus": {
+                "border-color": {
+                    "value": "{border.color.progressive.focus.value}"
+                },
+                "background-color": {
+                    "value": "{background.color.progressive.focus.value}"
+                },
+                "box-shadow": {
+                    "value": "{box-shadow.inset.focus.button-primary.value}"
+                },
+                "transition-timing-function": {
+                    "value": "{transition.timing-function.ease.value}"
+                },
+                "transition-property": {
+                    "value": "{transition.interaction.button-primary.focus.property.value}"
+                },
+                "transition-duration": {
+                    "value": "{transition.interaction.button.focus.duration.value}"
+                }
+            },
+            "disabled": {
```

Comment:
As far as I remember the disabled state is meant to look the same for all button ~~flavors~~ types, so its component-level tokens should not be nested within the type, but separately a level under `wikit-Button`. This is already reflected in the `Button.vue` css in that `&:disabled` is separated from the type selectors. It just wasn't fully implemented yet.

Reply:
There are specific disabled styles for the progressive button defined in Figma.
@SaiSan-WMDE are the button types supposed to look the same when they are disabled?

Reply:
Yes, all three button types look exactly the same when disabled (same applies to other button variants like quiet). I'm sorry to have caused this misunderstanding through the specs.